### PR TITLE
Add relabel config to plot durations for each gRPC method

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
For durations, stats reported by Envoy is enough to see.

image

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
Could you add some metrics samples here? (e.g. envoy_cluster_grpc_*_upstream_rq_time_bucket)
That helps to review the use of regular expressions.

**Comment by nakabonne** [comment]
@nghialv You're right.

Some examples here:

```
# TYPE envoy_cluster_grpc_service_webservice_WebService_GetInsightApplicationCount_upstream_rq_time histogram
envoy_cluster_grpc_service_webservice_WebService_GetInsightApplicationCount_upstream_rq_time_bucket{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api",le="0.5"} 0
envoy_cluster_grpc_service_webservice_WebService_GetInsightApplicationCount_upstream_rq_time_sum{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api"} 2770
envoy_cluster_grpc_service_webservice_WebService_GetInsightApplicationCount_upstream_rq_time_count{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api"} 6

# TYPE envoy_cluster_grpc_service_webservice_WebService_GetProject_upstream_rq_time histogram
envoy_cluster_grpc_service_webservice_WebService_GetProject_upstream_rq_time_bucket{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api",le="0.5"} 0
envoy_cluster_grpc_service_webservice_WebService_GetProject_upstream_rq_time_sum{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api"} 10.5
envoy_cluster_grpc_service_webservice_WebService_GetProject_upstream_rq_time_count{envoy_grpc_bridge_method="api",envoy_grpc_bridge_service="pipe",envoy_cluster_name="server-web-api"} 1
```

File: manifests/pipecd/values.yaml [hunk 1]
```diff
@@ -186,6 +186,31 @@ prometheus:
...
               regex: envoy-admin
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              target_label: method
+              regex: envoy_cluster_grpc_(.+)_(.+)_(.+)_(.+)_upstream_rq_time_bucket$
+              replacement: $4
+            - source_labels: [__name__]
+              target_label: __name__
+              regex: envoy_cluster_grpc_(.+)_(.+)_(.+)_(.+)_upstream_rq_time_bucket$
+              replacement: pipecd_requests_duration_bucket
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:197]: Should we add `envoy` to the name? for example, `pipecd_envoy_requests_`

Comment (nakabonne) [reply manifests/pipecd/values.yaml:197]: for sure, or `pipecd_gateway_requests_` seems to be better

Comment (nakabonne) [reply manifests/pipecd/values.yaml:197]: because this is a relabel_config for the pipecd-gateway job

Comment (nghialv) [reply manifests/pipecd/values.yaml:197]: Right. That is better. 👍

**Comment by nakabonne** [comment]
Thank you for looking over. Applied them!

**Pull request merged**